modules.json

```diff
@@ -366,5 +366,30 @@
             "list": "usercontribs",
             "ucuser": "Jimbo Wales"
     	}
-    }
-]
\ No newline at end of file
+    },
+	{
+		"filename": "blocks.py",
+		"docstring": "Demo of `Blocks` module: GET request to list recent blocks",
+		"endpoint": "https://en.wikipedia.org/w/api.php",
+		"params": {
+			"action": "query",
+			"format": "json",
+			"list": "blocks",
+			"bklimit": "3",
+			"bkprop": "id|user|by|timestamp|expiry|reason|range|flags"
+		}	
+	
+	},
+	{
+		"filename": "setlanguage.py",
+		"docstring": "Demo of `SetPageLanguage` module: POST request to change the language of a page",
+		"endpoint": "https://en.wikipedia.org/w/api.php",
+		"params": {
+			"action": "setpagelanguage",
+			"format": "json",
+			"pageid": "123",
+			"lang": "eu",
+			"token": "e17ac3bb8d7dd0c4f5e4fd1f84cd21095c76c2cc+\\"
```

Comment:
Information like tokens is sensitive information and one should never hard-code them in the code like you've added it here :) 

You will be programmatically obtaining a token and writing somewhat similar to `edit` module code for `setPageLanguage` module: https://github.com/wikimedia/MediaWiki-Action-API-Code-Samples/blob/master/python/edit.py

Reply:
okay, yeah I thought so! thanks a lot for the feedback! :)